modules/audio_coding/codecs/opus/opus_interface.h

```diff
@@ -526,6 +526,14 @@ int WebRtcOpus_PacketHasFec(const uint8_t* payload,
 int WebRtcOpus_PacketHasVoiceActivity(const uint8_t* payload,
                                       size_t payload_length_bytes);
 
+
+int16_t WebRtcOpus_SetMaxBandwidth(OpusEncInst* inst, int32_t max_bandwidth);
+int16_t WebRtcOpus_SetPredicationDisabled(OpusEncInst* inst, int32_t disable_prediction);
+int16_t WebRtcOpus_SetForceMode(OpusEncInst* inst, int32_t mode);
+int16_t WebRtcOpus_SetSignal(OpusEncInst* inst, int32_t signal);
+int16_t WebRtcOpus_SetApplication(OpusEncInst* inst, int32_t application);
```

Comment:
After measurements, it seems that we won't need to change these settings, especially at the lower bitrates we are working with, so maybe remove these changes?

Reply:
That is, prediction, application, signal, and forceMode.

Reply:
We don't need MaxBandwidth, either.  I think we just want SetBandwidth.  I'll remove them.